bookclub-backend/models/books.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const booksSchema = new mongoose.Schema({
+  gid: String,
+  img: String,
+  title: String,
+  authors: [{ type: String }],
```

Comment:
When just needing to give the `type` inside an array, you can just leave it as `[String]`. Please fix this for `/models/books.js` and `/models/users.js`